CooperationHullMainSite/Views/Shared/_Layout.cshtml

```diff
@@ -13,6 +13,14 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lexend:wght@100..900&display=swap" rel="stylesheet">
 
+    <!--Open Graph-->
+
+    <meta property="og:title" content="Cooperation Hull" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="\assets\img\Cooperation Hull Circle Logo 2.png" />
```

Comment:
Small thing, but we should get a version of this logo that has some breathing room around the edges